converter/src/DelphesEDM4HepConverter.cc

```diff
@@ -189,6 +190,10 @@ namespace k4SimDelphes {
     //UserData for overflowing information
     auto* pathLengthCollection = createCollection<podio::UserDataCollection<float>>(branch + "_L");
 
+    // user data to define magnetic field (fill once per event)
+    auto* magFieldCollection = createCollection<podio::UserDataCollection<float>>("B");
```

Comment:
Should we give this a slightly more descriptive name? I would at least add a `z` to indicate the direction in this case.

Reply:
indeed, thanks for the comment, done!